packages/generator-common/src/sdk-metadata/generation-and-usage.ts

```diff
@@ -1,19 +1,27 @@
-import { InstructionWithText } from './sdk-metadata-types';
+import { InstructionWithTextAndHeader } from './sdk-metadata-types';
 
 export function getGenerationSteps(
   installCommand: string,
   generateCommand: string,
-  generationDocLink: string
-): InstructionWithText {
+  generationDocLink: string,
+  apiType: 'OData' | 'OpenApi'
+): InstructionWithTextAndHeader {
   return {
+    header: 'Installation & Generation Steps',
     instructions: `<ul>
-<li>Download the API specification to your local computer.</li>
-<li>Install the generator <code>${installCommand}</code></li>
+<li>Install <a href="https://docs.npmjs.com/downloading-and-installing-node-js-and-npm">Node and npm.</a>. The use of the LTS version is preferred.</li>
+<li>Download the API specification from Overview -> API Specification section. ${
+      apiType === 'OData' ? 'Chose EDMX format.' : 'Chose JSON format.'
+    }</li>
+<li>Generate a typed client by running <code>${installCommand}</code> to install the code generator.</li>
 <li>Execute the generator <code>${generateCommand}</code></li>
-</ul>
-The steps above will generate a basic TypeScript client. For additional options like transpiling to JavaScript etc. visit our <a href="${generationDocLink}">documentation.</a>`,
-    text: 'Follow the following generation steps to generate the client.'
+<li>You should now find generated classes in the output directory specified in step 4.</li>
+<li>By default we generate TypeScript only. If you need to transpile it to JavaScript check the <a href="https://sap.github.io/cloud-sdk/docs/js/features/odata/generate-odata-client#options>"code generator CLI parameters in the documentation.</a></li>
+<li>In case of any issues carefully check your configuration and refer to our <a href="https://sap.github.io/cloud-sdk/docs/js/features/odata/generate-odata-client">extended typed client generation manual.</a></li>
```

Comment:
```suggestion
<li>In case of any issues, carefully check your configuration and refer to our <a href="https://sap.github.io/cloud-sdk/docs/js/features/odata/generate-odata-client">extended typed client generation manual</a>.</li>
```